hs/src/Reach/Eval/Core.hs

```diff
@@ -3680,11 +3680,13 @@ evalPrim p sargs =
       ensure_modes [SLM_ConsensusStep, SLM_ConsensusPure] "remote ALGO"
       metam <- mustBeObject =<< one_arg
       metam' <- mapM (ensure_public . sss_sls) metam
-      let actual = M.keysSet metam'
-      let valid = S.fromList $ [ "fees", "accounts", "assets", "addressToAccount", "apps", "onCompletion", "strictPay", "rawCall" ]
-      unless (actual `S.isSubsetOf` valid) $ do
+      let keys = M.keysSet metam'
+      let validKeys = S.fromList $ [ "fees", "accounts", "assets", "addressToAccount", "apps",
+                                     "onCompletion", "strictPay", "rawCall",
+                                     "simulationValues" ]
```

Comment:
I think "simulationValues" is a confusing interface and it should be `sim_netRecv` ... etc